object_labels:
  - 5
  - 6
  - 7
  - 9
  - 10
  - 18

